src/tensor_ops/concat/cuda_kernel.rs

```diff
@@ -39,6 +39,7 @@ impl<E: Dtype + CudaTypeName> super::ConcatKernel<E> for Cuda {
             let src = BWD_KERNEL.replace("$Ty", E::NAME);
             let opts = CompileOptions {
                 arch: Some(env!("CUDA_COMPUTE_CAP")),
+                include_paths: vec!["/usr/include".to_string()],
```

Comment:
This is done a couple places to include "cuda_fp16.h". What is a better string for this? Could we use cuda toolkit root as include directory? Might require some updates to cudarc to set that env variable